source/introduction.txt

```diff
@@ -127,10 +127,10 @@ logic deals with sensitive user data or interacting with external services. Apps
 also need to respond in real time when something happens in the app or in an external service.
 
 {+backend+} lets you define serverless JavaScript functions called {+backend-short+} Functions.
-You can use Realm Functions for the following common backend use cases:
```

Comment:
Line 106 above is "a {+backend-short+} schema" which expands to "a App Services Schema" - should be "an App Services schema"